tests/python/frontend/onnx/test_forward.py

```diff
@@ -1282,6 +1282,47 @@ def verify_batch_matmul(a_shape, b_shape, out_shape, convert_config=None):
     )
 
 
+@tvm.testing.parametrize_targets
+def test_matmulinteger16(target, dev):
+    def verify_matmulinteger16(a_shape, b_shape, out_shape):
+        a_dtype = "int16"
+        b_dtype = "int16"
+        low = -10
+        high = 10
```

Comment:
Is there a particular reason for making the bounds -10 ... 10?

Reply:
Since the op can be used for signed/unsigned integers, the range is kept as [-10, 10] to cover common inputs.

Reply:
Would it makes sense to add a comment saying that these are semi-arbitrary values? It might help readers avoid looking for a deeper meaning for why these particular numbers were chosen.

Reply:
Or maybe use `np.iinfo(np.int16).min` and `...max` as the bounds?

If the goal is to increase the random discovery of latent bugs, and we don't know the _specific_ values at which corner-cases happen, perhaps it makes sense to at least range over all possible values.

I'm not sure what the best strategy is here, but I'm wondering if this is a step in the right direction.